README.md

```diff
@@ -251,20 +251,11 @@ type spaHandler struct {
 // file located at the index path on the SPA handler will be served. This
 // is suitable behavior for serving an SPA (single page application).
 func (h spaHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-    // get the absolute path to prevent directory traversal
-	path, err := filepath.Abs(r.URL.Path)
-	if err != nil {
-        // if we failed to get the absolute path respond with a 400 bad request
-        // and stop
-		http.Error(w, err.Error(), http.StatusBadRequest)
-		return
-	}
-
-    // prepend the path with the path to the static directory
+    // Join internally call path.Clean to prevent directory traversal
 	path = filepath.Join(h.staticPath, path)
```

Comment:
`path` is never initialized

Reply:
thanks for pointing it out. I have fixed the issue.